runtime/j9vm/jvm.c

```diff
@@ -643,7 +669,7 @@ captureCommandLine(void)
 #elif defined(LINUX) /* defined(AIXPPC) */
 	int fd = open("/proc/self/cmdline", O_RDONLY, 0);
 
-	if (fd >= 0) {
+	if (0 <= fd) {
```

Comment:
A constant is fine on the right of `>=` (it's only `==` and `!=` where the danger is higher that someone writes `=` instead, in which case the compiler would complain for things like `0 = fd`).